devops/linux/docker/deploy-local.sh

```diff
@@ -0,0 +1,162 @@
+# #!/bin/bash
+# Set JEMPI_HOME environment variable
+cd ../../..
+export JEMPI_HOME=$(pwd)
+export JAVA_VERSION=17.0.8.1-tem
+echo "Setting JEMPI_HOME to: $JEMPI_HOME"
+#JEMPI_CONFIGURATION_PATH=$JEMPI_HOME/JeMPI_Apps/JeMPI_Configuration/ethiopia/config-ethiopia-3.json
+JEMPI_CONFIGURATION_PATH=$JEMPI_HOME/JeMPI_Apps/JeMPI_Configuration/reference/config-reference.json
+
+# Display menu options
+echo "Select an option for local deployment:"
+echo "1. Deploy JeMPI from Scratch (With all installations...)."
+echo "2. Build and Reboot."
+echo "3. Restart JeMPI."
+echo "4. Down the JeMPI."
+echo "5. Destroy JeMPI (This process will wipe all data)."
+
+
+
+
+# Prompt user for choice
+read -p "Enter your choice (1-5): " choice
+
+
+install_docker() {
+    if command -v docker &> /dev/null; then
+        echo "Docker is already installed."
+    else
+        # Install Docker
+        echo "Installing Docker... "
+        sudo apt-get update
+        sudo apt-get install -y docker.io
+
+        # Add your user to the docker group to run Docker without sudo
+        sudo usermod -aG docker $USER
+
+        echo "Docker has been installed."
+    fi
+}
+
+# Function to install SDKMAN, Java, Maven, and SBT
+install_sdkman_and_java_sbt_maven() {
+    # Install SDKMAN
+    echo "Installing SDKMAN... "
+    curl -s "https://get.sdkman.io" | bash
+
+    # Initialize SDKMAN
+    source "$HOME/.sdkman/bin/sdkman-init.sh"
+
+    # Install Java, Maven, and SBT using SDKMAN
+    echo "Installing Java : $JAVA_VERSION ... "
+    sdk install java $JAVA_VERSION
+    sdk use java $JAVA_VERSION
+
+    echo "Installing maven... "
+    sdk install maven
+
+    echo "Installing sbt... "
+    sdk install sbt
+}
+run_enviroment_configuration_and_helper_script(){
+    # Navigate to environment configuration directory
+    echo "Navigate to environment configuration directory"
+    cd $JEMPI_HOME/devops/linux/docker/conf/env/
+    source $JEMPI_HOME/devops/linux/docker/conf/env/create-env-linux-low-1.sh
```

Comment:
Are you not considering the high configuration `devops/linux/docker/conf/env/create-env-linux-high-1.sh`. Should you not pass an option that allows you to choose this?